src/backend/dialog/dialog.ts

```diff
@@ -52,4 +53,22 @@ function showDialogBoxModalAuto(props: {
   }
 }
 
-export { showDialogBoxModalAuto }
+type NotifyType = {
+  title: string
+  body: string
+}
+
+function notify({ body, title }: NotifyType) {
+  if (Notification.isSupported() && !isSteamDeckGameMode) {
+    const mainWindow = BrowserWindow.getAllWindows()[0]
```

Comment:
we can use `getMainWindow` from the `main_window` module here to hide the implementation